tests/testAPI.sh

```diff
@@ -41,6 +44,16 @@ function check_auth_token() {
 	fi
 }
 
+function get_bearer_token() {
+        if [[ $CLUSTER_TYPE == "openshift" ]]; then
+		check_tool_existance "oc"
+                AUTH_TOKEN=$(oc whoami --show-token)
+        elif [[ $CLUSTER_TYPE == "ICP" ]]; then
+                check_tool_existance "cloudctl"
+                AUTH_TOKEN=$(cloudctl tokens | grep "Bearer"  | cut -d' ' -f5)
+        fi
```

Comment:
What happens if the cluster type is neither openshift nor ICP ? Please use a validation function to ensure that the CLUSTER_TYPE is valid